airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1956,23 +1956,85 @@ definitions:
 interpolation:
   variables:
     - title: config
-      description: The connector configuration. The config's keys match the keys defined in the spec.
+      description: The connector configuration. The object's keys are the same as the the keys defined in the connection specification.
+      type: object
+      examples:
+        - start_date: 2010-01-01
+          api_key: "*****"
     - title: headers
-      description: The HTTP headers from the last response received from the API.
+      description: The HTTP headers from the last response received from the API. The object's keys are the header names from the response.
+      type: object
+      examples:
+        - Server: nginx
+          Date: Mon, 24 Apr 2023 20:17:21 GMT
+          Content-Type: application/json
+          Content-Length: "420"
+          Connection: keep-alive
+          referrer-policy: strict-origin-when-cross-origin
+          x-content-type-options: nosniff
+          x-ratelimit-limit: "600"
+          x-ratelimit-remaining: "598"
+          x-ratelimit-reset: "39"
     - title: last_records
       description: List of records extracted from the last response received from the API.
+      type: list
+      examples:
+        - - name: "Test List: 19"
+            id: 0236d6d2
+            contact_count: 20
+            _metadata:
+              self: https://api.sendgrid.com/v3/marketing/lists/0236d6d2
+          - name: List for CI tests, number 30
+            id: 041ee031
+            contact_count: 0
+            _metadata:
+              self: https://api.sendgrid.com/v3/marketing/lists/041ee031
+    - title: next_page_token
+    - description: Object describing the token to fetch the next page of records. The object has a single key "next_page_token".
```

Comment:
Remove the dash here, otherwise theres an object too much

Reply:
done